cms/static_src/javascript/main.js

```diff
@@ -1 +1 @@
-import '../sass/main.scss';
+import '../sass/main.scss'
```

Comment:
Query: should the changes for megalinter be updating js and scss files, as we already have linting set up for these in the form of eslint and stylelint?